src/App.vue

```diff
@@ -523,15 +526,15 @@ export default {
     async fetchDraftAppData (): Promise<void> {
       this.nameRequestInvalidType = null // reset for new fetches
 
-      const draft = await LegalServices.fetchDraftApp(this.tempRegNumber)
+      const application = await LegalServices.fetchDraftApp(this.tempRegNumber)
 
-      // Handle Draft filings
-      this.storeDraftApp(draft)
+      // handle draft application
+      this.storeDraftApp(application)
 
-      // if the draft has a NR, load it
-      if (this.localNrNumber) {
+      // if this app is a task (not a filing), and it has a NR, load it
+      if (this.isAppTask && this.localNrNumber) {
```

Comment:
This is the fix for 18736 (which was merged into a release branch and deployed to Prod already).